src/test/user.spec.js

```diff
@@ -37,7 +42,18 @@ describe('Test user login, signup and account verification', () => {
         });
     });
 
-    it('it should throw an error when user with an unverified account wants to signup', done => {
+    it('should mock a sendgrid api request', async () => {
+      const secret = process.env.SENDGRID_API_KEY;
+      sgMail.setApiKey(secret);
+
+      nock('https://api.sendgrid.com')
+        .post('/api/mail.send.json')
+        .reply(200, {
+          message: 'success'
+        });
+    });
+
```

Comment:
I noticed you just mocked the sendgrid API but there is no assertion to test the expected response. You may have to look at [this](https://github.com/sendgrid/sendgrid-nodejs/issues/141#issuecomment-63570373) for some insight.